tests/torch/test_sanity_third_party.py

```diff
@@ -204,7 +204,8 @@ def test_glue_distilbert_eval(self, temp_folder):
 
     @pytest.mark.dependency(depends=['install_trans'], name='lm_train')
     def test_lm_train(self, temp_folder):
-        com_line = "examples/pytorch/language-modeling/run_clm.py --model_name_or_path gpt2" \
+        # GPT2 is loaded via torch.frombuffer which is not available in torch==1.9.1 yet
+        com_line = "examples/pytorch/language-modeling/run_clm.py --model_name_or_path distilgpt2" \
```

Comment:
GPT2 is loaded via `safetensor` functionality that uses `torch.frombuffer`. This function appears since torch1.10, but we support torch==1.9.1 only.